test/e2e/helm/kuma_helm_deploy_global_universal_mode_and_zone.go

```diff
@@ -0,0 +1,160 @@
+package helm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/random"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+
+	api_server "github.com/kumahq/kuma/pkg/api-server"
+	"github.com/kumahq/kuma/pkg/config/core"
+	. "github.com/kumahq/kuma/test/framework"
+	"github.com/kumahq/kuma/test/framework/deployments/postgres"
+	"github.com/kumahq/kuma/test/framework/deployments/testserver"
+)
+
+func ZoneAndGlobalInUniversalModeWithHelmChart() {
+	var clusters Clusters
+	var c1, c2 Cluster
+	var global, zone ControlPlane
+
+	BeforeAll(func() {
+		var err error
+		clusters, err = NewK8sClusters(
```

Comment:
sigh. How is this still here? I thought I removed `New*Clusters`. I remember I was doing it, maybe I did not create a PR in the end. Please use `NewK8sCluster`, don't use Clusters. It's not a good abstraction, we should get rid of it.